client/src/pages/Review/Course.tsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+
+import { classnames } from "../../Utilities";
+import CourseInfo from "../../components/CourseInfo";
+import Tabs, { Tab } from "../../components/Tabs";
+import { Assessment, Course } from "../../logic/icsGen";
+import EditAssessment from "../../components/EditAssessment";
+
+import {default as AssessmentPanel} from "./Assessment";
```

Comment:
Rename Assessment.tsx to AssessmentPanel.tsx for consistency

Reply:
Do you want other files like `Course.tsx` to be renamed too?

Reply:
Yeah I think it's best if we stay consistent that way. Assessment just stood out because I expected it to be the individual Assessment card and then it wasn't. Better yet make it Assessment**s**Panel.tsx